src/CoreWCF.WebHttp/src/CoreWCF/OpenApi/Attributes/OpenApiPropertyAttribute.cs

```diff
@@ -20,5 +20,55 @@ public sealed class OpenApiPropertyAttribute : Attribute
         /// Whether the property is required or not.
         /// </summary>
         public bool IsRequired { get; set; }
+
+        /// <summary>
+        /// Minimum length of the property.
+        /// </summary>
+        public int MinLength
+        {
+            get
+            {
+                return _minLength;
+            }
+            set
+            {
+                _minLength = value;
+                MinLengthSet = true;
+            }
+        }
+
+        /// <summary>
+        /// Maximum length of the property.
+        /// </summary>
+        public int MaxLength
```

Comment:
What do you think about defining MaxLength and MinLength as nullables to know if a value was set? So define them like this:
```c#
public int? MaxLength { get; set; }
```

Reply:
That was the initial route I went down, but I got a "'MaxLength' is not a valid named attribute argument because it is not a valid attribute parameter type".

Reply:
I just thought of another way to do this. Don't use GetCustomAttributes, use GetCustomAttributesData instead. That doesn't give you an instance of the attribute, it gives you an instance of CustomAttributeData. This is a representation of how the attribute was used in the original code. If has a NamedArguments list which only contains the arguments specified on the attribute. If they didn't specify MaxLength, there won't be an entry in NamedArguments for MaxLength. It's a lot cleaner way of handling this.

Reply:
Oooh that is clever. Changes have been pushed.